<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuWindow</class>
 <widget class="QDialog" name="MenuWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>241</width>
     <height>481</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-40</y>
     <width>1031</width>
     <height>641</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>img/gradient.jpg</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>40</y>
     <width>241</width>
     <height>481</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>40</y>
     <width>241</width>
     <height>481</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>pushButton_3</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>MenuWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>774</x>
     <y>448</y>
    </hint>
    <hint type="destinationlabel">
     <x>872</x>
     <y>712</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
